js/main.js

```diff
@@ -447,3 +447,16 @@ function setupShareCopy(){
         Materialize.toast('Failed to copy share link!', 1000);
     });
 }
+
+function clearAll(){
+    console.log("remove all");
+
+    player.stopVideo();
+    var id;
+    for(var i=0; i<playlist.length; i++){
+        id=playlist[i].id.videoId;
+        $("#playlist #"+id).remove();
```

Comment:
If we aren't going for the animation, `$('#playlist li').remove()` would look much cleaner, instead of the for loop.